dataset.proto

```diff
@@ -22,6 +22,7 @@ message DataSet {
   message Malicious {
     int32 category   = 1;
     string signature  = 2;
+    bool verdict = 3;
```

Comment:
Would it make more sense to call the field "malicious" then? Maybe not because then it a check would look like:
```
if myDS.Malicious.malicious { do something }
```

On second thought, I like "verdict" better.